examples/data-qc-checks/check-for-missing-instrument-fields/1-create-instrument-properties.sql

```diff
@@ -5,6 +5,11 @@
 -- See the following page for further details:
 -- https://support.lusid.com/knowledgebase/article/KA-01702/en-us
 -- ===============================================================
+
+@x = use Sys.Admin.SetupView
+--provider=Views.DQ_CreateInstrumentProperties
```

Comment:
Looks good, but I wonder if it might be cleaner to follow more of a 3 step structure with something like:
`Views.DataQuality.CreateInstrumentProperites` rather than using the `_`. 

We could then use this further to have a structure for the demo views in format:
Views
    --> SomeUsecase
        ---> SomeView

A bit of a nitpick, so will leave with you.